about.html

```diff
@@ -0,0 +1,116 @@
+            <div class="menubar" id="rotateLeft"></div>
+        </a>
+        <a href="./index.html" class="logo headerlogo"></a>
+        <ul class="menuLinks">
+            <li><a href="./index.html">Main</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#events">Events</a></li>
+            <li><a href="#partners" class="partnerlink">Partners</a></li>
+        </ul>
+
+    </header>
+    <main>
+        <section id="aboutpage">
+            <section class="welcome-section about-welcome-section">
+                <p class="hello" id="abouthello">"Hello Nigeria!"</p>
+                <h1 class="abouth" id="abouth">"MADE IN LAGOS" WIZKID FEST 2023</h1>
+                <p class="welcomeP aboutP">The Music Fest brings together the community <br> of experts,professionals
+                    and
+                    talented
+                    musicians home and away <br>we host the Music Fest concert in a <br> different country every two
+                    years and this year we are bringing it home!.
+                    This
+                    year, we hope to expand our invitation list including <br> organizations and individuals.</p>
+                <p class="welcomeP welcomeP2 aboutP1">Please contact us via Email for any further questions about the
+                    Made
+                    in Lagos Music Fest 2023!</p>
+                <a href="#" id="email">lanzzd191@gmail.com</a>
+            </section>
+            <section class="praises" id="praises">
+                <div class="heading">
+                    <h3>"Made in Lagos" Wizkid Fest 2023 Logo</h3>
+                    <div class="redline"></div>
+                </div>
+                <div class="logodiv">
+                    <p class="welcomeP" id="praisesP">The logo of CC Global Summit was decided through the logo
+                        competition
+                        from
+                        8. June to 7. July</p>
+                    <a href="./index.html" class="logo aboutlogo mobilelogo"></a>
+                </div>
+                <section class="previousSummitCard">
+                    <div class="heading">
+                        <h3>See the past "Made in Lagos" Music Fests</h3>
+                        <div class="redline"></div>
+                    </div>
+                    <p>Take a look at the two "Made in Lagos" Music Fest which took place in Nigeria and the United
+                        Kingdom</p>
+                    <ul class="reference">
+                        <li class="fst">
+                            <h2>2021</h2>
+                            <p>"Made in Lagos" Music Fest in Victoria Island</p>
+                        </li>
+                        <li class="lst">
+                            <h2>2018</h2>
+                            <p>"Made in Lagos" Wizkid Fest at the O2 Arena</p>
+                        </li>
+                    </ul>
```

Comment:
- [ Required ] Great job implementing the changes by a previous reviewer on this section, however, if you look closely you'll notice that on the `Behance` design this part of the design has a `reddish` transparent background covering the two images. It would be better if you implement this to make your design Mirror the `Behance` design. Please see the screenshot below for more details on this.

- [ Required ] Also consider centering the text above the images as shown on the `Behance` design.

## Your design
<img src="https://user-images.githubusercontent.com/16392046/217850106-a9b9ac67-0ca8-4d16-a9fa-0d56a58bb065.png" width="375px" />

## Behance design
<img src="https://user-images.githubusercontent.com/16392046/217850404-8db7dac0-7acb-486f-bca4-bd338af923a6.png" width="375px" />